src/corehost/cli/test/nativehost/nativehost.cpp

```diff
@@ -67,6 +68,34 @@ int main(const int argc, const pal::char_t *argv[])
             return 1;
         }
     }
+#if defined(_WIN32)
+    else if (pal::strcmp(command, _X("comhost")) == 0)
+    {
+        // args: ... <scenario> <activation_count> <comhost_path> <clsid>
+        if (argc < 6)
+        {
+            std::cerr << "Invalid arguments" << std::endl;
+            return -1;
+        }
+
+        const pal::char_t *scenario = argv[2];
+        int count = pal::xtoi(argv[3]);
+        const pal::string_t comhost_path = argv[4];
+        const pal::string_t clsid_str = argv[5];
+
+        bool success = false;
+        if (pal::strcmp(scenario, _X("synchronous")) == 0)
+        {
+            success = comhost_test::synchronous(comhost_path, clsid_str, count);
+        }
+        else if (pal::strcmp(scenario, _X("concurrent")) == 0)
+        {
+            success = comhost_test::concurrent(comhost_path, clsid_str, count);
+        }
+
+        return success ? 0 : 1;
```

Comment:
The standard actually defines these return values: `EXIT_SUCCESS` and `EXIT_FAILURE`.